### DISCUSSION TITLE
How do we do custom props for nativewind now that `styled` is gone in `v4`

### DISCUSSION THREAD

**Discussion by lakardion** [nativewind/nativewind#858] [Q&A]
So I used to have this very useful component:

```
export const ScrollViewWind = styled(ScrollView, {
  props: {
    contentContainerStyle: true,
  },
})
```

Now that `styled` is gone, how can I achieve the same? 

So that I can pass tailwind classes to the `ScrollViewWind` component rather than the style object to `ScrollView`

**Comment by patogordo** [comment]
Any update?

**Comment by idlework** [comment]
For React Native components this already is implemented. For example, with ScrollView the `contentContainerStyle` is mapped to `contentContainerClassName`.


But if you want to style custom or third party components you can use the [remapProps](https://www.nativewind.dev/api/remap-props) utility for this.

**v2 Styled**
``` javascript
const StyledScrollView = styled(ThirdPartyScrollView, {
  props: {
    contentContainerStyle: true,
  },
})
```

**v4 Remap props**
``` javascript
const RemappedScrollViewWind = remapProps(ThirdPartyScrollView, {
 contentContainerClassName: "contentContainerStyle",
})
```

**Reply by sairam4123** [reply]
How to use this.. do we need to use the RemppedScrollViewWind everywhere we used that thirdparty component?

**Comment by danstepanov** [comment]
`styled()` is back in v5
